app/views/books/_form.html.slim

```diff
@@ -0,0 +1,51 @@
+= simple_form_for @book do |f|
```

Comment:
you are passing `@book` as a variable `book` into that partial, but are still using `@book`

Reply:
done